main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	// "net/url"
+	"os"
+)
+
+func main() {
+	var page int
+	page = 1
+	// params := url.Values{}
+	// params.Add("page", page)
+	for i := 0; i <155; i++{
+		url := fmt.Sprint("https://www.balldontlie.io/api/v1/players?page=", page)
+		// resp, err := http.Get("https://www.balldontlie.io/api/v1/players"+params.Encode())
+		resp, err := http.Get(url)
+		if err != nil {
+		log.Fatalln(err)
+		}
+
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+		log.Fatalln(err)
+		}
+
+		// sb := string(body)
+		// fmt.Printf("%T", sb)
+		// fmt.Print(sb)
+
+
+		type Player struct {
```

Comment:
Lets change this up and instead make two structs. One for just the Player itself
```
type Player struct {
    Id int ....
    ....
}
````

And another for the API response itself, the envelope for all the players. Something like 
```
type Response struct {
    Data [] Player
}
```

For the `Player` struct, lets stick this outside the `main()` function and at what we would call `top lavel`. That way it can be available to other functions, that I'm going to ask you to write.